some.js

```
@@ -0,0 +1,22 @@
+// example.js
+
+const myfunction= ()=> {
+console.log("Hello, world!")
+return 42;
+} 
+
+myfunction( );
+
+
+// Mocha Test with issues
+const assert = require('assert');
```

ESLint

[error] 'require' is not defined.

(no-undef)